okhttp-android-support/src/test/java/com/squareup/okhttp/internal/huc/ResponseCacheTest.java

```diff
@@ -1113,10 +1113,34 @@ public void assertCookies(CookieManager cookieManager, URL url, String... expect
   }
 
   @Test public void cachePlusRange() throws Exception {
```

Comment:
I'd prefer to avoid suggesting we support any range caching in our tests because the built-in Cache class can't handle ranges. They're a lot of work. For now can we just avoid offering support for HTTP_PARTIAL ?